ompi/mca/pml/ucx/pml_ucx.c

```diff
@@ -225,7 +225,7 @@ int mca_pml_ucx_init(void)
     if (ompi_mpi_thread_multiple && (attr.thread_mode != UCS_THREAD_MODE_MULTI)) {
         /* UCX does not support multithreading, disqualify current PML for now */
         /* TODO: we should let OMPI to fallback to THREAD_SINGLE mode */
-        PML_UCX_ERROR("UCP worker does not support MPI_THREAD_MULTIPLE");
+        PML_UCX_VERBOSE(1, "UCP worker does not support MPI_THREAD_MULTIPLE");
```

Comment:
so if UCX does not support multi-threading, another PML would be used?
Maybe we should keep using PML_UCX, but report a lower thread-support level to the SHMEM application?

Reply:
i don't think this is the behavior we want

Reply:
ok, fixed: it will fallback into single thread mode